CONTRIBUTING.md

```diff
@@ -29,3 +29,99 @@ Recommendations and requirements for how to best contribute to Parquet. We striv
 ### License
 By contributing your code, you agree to license your contribution under the terms of the APLv2:
 https://github.com/apache/parquet-format/blob/master/LICENSE
+
+### Additions/Changes to the Format
+
+Note: This section applies to actual functional changes to the
+specification. Fixing typos, grammar, and clarifying concepts
+that would not change the semantics of the specification can
+be done as long a comitter feels comfortable to merge them. When
+in doubt starting a discussion on the dev mailing list is 
+encouraged.
+
+The general steps for adding features to the format are as follows:
+
+1. Discuss changes on on the developer mailing list (dev@parquet.apache.org).  Often times it is helpful to link to a draft pull request to make the discussion concrete. This step is complete when there lazy consensus.
```

Comment:
You seem to be sometimes line wrapping and sometimes you're not. Should we have a consistent line wrapping style?

Reply:
Apologies, just pushed a commit to make it more consistent (used line wrapping and reflowed text)